src/plugins/plugin.title.js

```diff
@@ -22,19 +20,19 @@ defaults._set('global', {
 /**
  * IMPORTANT: this class is exposed publicly as Chart.Legend, backward compatibility required!
  */
-var Title = Element.extend({
-	initialize: function(config) {
+class Title extends Element {
+	initialize(config) {
 		var me = this;
 		helpers.extend(me, config);
 
 		// Contains hit boxes for each dataset (in dataset order)
 		me.legendHitBoxes = [];
-	},
+	}
 
 	// These methods are ordered by lifecycle. Utilities then follow.
 
-	beforeUpdate: noop,
-	update: function(maxWidth, maxHeight, margins) {
+	beforeUpdate() {}
```

Comment:
Each of these will be a new function, right? So we get a lot of empy functions.

Reply:
Yes, though I wouldn't expect there'd be a performance impact since it's only in `title` and `legend` we don't have many instances of